pkg/tfvars/baremetal/baremetal.go

```diff
@@ -66,6 +67,7 @@ func TFVars(libvirtURI, apiVIP, imageCacheIP, bootstrapOSImage, externalBridge,
 		driverInfo := accessDetails.DriverInfo(credentials)
 		driverInfo["deploy_kernel"] = fmt.Sprintf("http://%s/images/ironic-python-agent.kernel", net.JoinHostPort(imageCacheIP, "80"))
 		driverInfo["deploy_ramdisk"] = fmt.Sprintf("http://%s/images/ironic-python-agent.initramfs", net.JoinHostPort(imageCacheIP, "80"))
```

Comment:
Do we still need the IPA references here?

Reply:
Yes. ISO is only supported for virtual media.

Reply:
These are rhcos live images, so yes, we need them.